dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -281,6 +313,41 @@ def redirect_post(key_params)
       end
     end
 
+    def render_csf_survey(survey_name, workshop)
```

Comment:
Maybe we should mark this method `private` to make it more obvious that it's a helper, not itself a route.

Reply:
Will do

Reply:
This method is actually under `protected` section with other methods. Do we need to be stricter for this one?

Reply:
No, that's fine.  Thanks! :+1: